src/CXX/Python3/cxxextensions.c

```diff
@@ -42,13 +42,23 @@ extern "C"
 {
 #endif
 
+#if PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION >= 13
```

Comment:
Is the 13 here correct?

Reply:
Yes, it is. I fetched the latest code changes from upstream: https://sourceforge.net/p/cxx/code/459/